android/src/main/java/org/reactnative/camera/CameraViewManager.java

```diff
@@ -201,25 +201,16 @@ public void setTextRecognizing(RNCameraView view, boolean textRecognizerEnabled)
     view.setShouldRecognizeText(textRecognizerEnabled);
   }
 
-  /**---limit scan area addition---**/
   @ReactProp(name = "rectOfInterest")
   public void setRectOfInterest(RNCameraView view, ReadableMap coordinates) {
     if(coordinates != null){
-      float x = (float) coordinates.getDouble("x");
-      float y = (float) coordinates.getDouble("y");
-      float width = (float) coordinates.getDouble("width");
-      float height = (float) coordinates.getDouble("height");
-      view.setRectOfInterest(x, y, width, height);
-    }
-  }
-
-  @ReactProp(name = "cameraViewDimensions")
-  public void setCameraViewDimensions(RNCameraView view, ReadableMap dimensions) {
-    if(dimensions != null){
-      int cameraViewWidth = (int) dimensions.getDouble("width");
-      int cameraViewHeight = (int) dimensions.getDouble("height");
-      view.setCameraViewDimensions(cameraViewWidth, cameraViewHeight);
+      int dataWidth = (int) coordinates.getInt("dataWidth");
+      int dataHeight = (int) coordinates.getInt("dataHeight");
+      int left = (int) coordinates.getInt("left");
+      int top = (int) coordinates.getInt("top");
+      int width = (int) coordinates.getInt("width");
+      int height = (int) coordinates.getInt("height");
+      view.setRectOfInterest(dataWidth, dataHeight, left, top, width, height);
     }
   }
-  /**---limit scan area addition---**/
```

Comment:
`dataWidth` and `dataHeight` should be exposed to the javascript developer with a `getter`. 
`cameraView.getPreviewSize` returns the `dataWidth` and `dataHeight`, while `cameraView.getPictureSize` returns the image height and width.